### DISCUSSION TITLE
Target class [Controller Name] does not exist.

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5670] [Q&A]
I just create Target class NotificationController as laravel creation, the related model and views and add new nav in AppServiceProvider like that :

Nav::extend(function ($nav) {
            $nav->content('Notification')
                    ->url('notification')
                    ->icon('shopping-cart');
        });
        
and the route in web route is : Route::get('cp/notification', 'NotificationController@index')->name('notification.index');

but it shows me Target class [NotificationController] does not exist. How to fix this please.

when i edit the route to be : Route::get('notification', 'NotificationController@index')->name('notification'); it route but show me on empty page that Page Not Found ??????

**Comment by edalzell** [comment]
Can we see the actual error stack?

**Comment by jasonvarga** [comment]
You need to add the namespace.

Instead of just `'NotificationController@index'` you should do `'App\Http\Controllers\NotificationController@index'` (or whatever the appropriate namespace is for your class)
